### DISCUSSION TITLE
Unsure how to use

### DISCUSSION THREAD

**Discussion by AJeschor** [ChocolateLoverRaj/pam-any#9] [Q&A] [answered]
While I love the idea and wish to implement it, I lack the understanding on how to do so. I would greatly appreciate a more detailed or easy to understand set of instructions on how to use this

**Comment by ChocolateLoverRaj** [comment, marked as answer]
You can check https://github.com/ChocolateLoverRaj/pam-any/blob/2ffac02d617bf1d6bc9368b551d81b113cb6cdc2/pam-any#L1 as an example. Basically, the parameter to the pam-any module is a JSON object where the modules object takes in a name of a different pam module as keys and a description as the value. Let me know if I should explain in more detail.
